web/src/components/checkbox/index.tsx

```diff
@@ -50,19 +50,23 @@ export const Checkbox: React.FC<CheckboxProps> = props => {
     isChecked: state.isSelected,
   })
   const hasLabel = isLabelCheckbox(props)
+  const labelId = genUid()
```

Comment:
seems like the `labelId` gets re-generated every time I click the checkbox, why?

Reply:
Right, I need to use memoization. Thanks!